concourse/pipelines/create-cloudfoundry.yml

```diff
@@ -2996,6 +3001,8 @@ jobs:
           - name: cf-manifest
         outputs:
           - name: rotated-cf-secrets
+        params:
+          UAA_ENDPOINT: "https://uaa.((system_domain))"
```

Comment:
this should be `"https://uaa.((system_dns_zone_name))"`. Fix commit: 4041c8719c2b2dbfec4c2038e815e5ed557efb65